Podfile

```diff
@@ -259,7 +261,21 @@ pre_install do |installer|
   end
 end
 
+
+# Configure your macCatalyst dependencies
+catalyst_configuration do
```

Comment:
cc @AliSoftware on the cocoapods changes

Reply:
This looks like it aligns with the `cocoapods-catalyst-support` plugin usage, so no particular concerns from my side. 

Too bad they don't support Mac – I could see that being really nice!

Reply:
Thx for the ping.

I've never used that `cocoapods-catalyst-support` plugin before, this seems neat!
The configuration used here looks good to me from what I understand from what's expected from that plugin.

PS: Only suggestion I'd make is to unify the use of tabs vs spaces in those lines 😛  otherwise it looks like an indentation mismatch especially here on GitHub between lines using a tab and the ones using spaces. The rest of the file seems to use spaces everywhere so I suggest to use that on those comment lines L267-L270 + L272 🙃

Reply:
Indentation fixed in 3280fd85337ec5ef154aa46a99d29d521a8aea04 😊